src/api/app/models/kiwi/image.rb

```diff
@@ -157,6 +158,15 @@ def self.binaries_available(project, use_project_repositories, repositories)
 
   private
 
+  def kiwi_indentation(xml)
```

Comment:
You are basing this calculation in the space between the image tag and the first child inside it... what if this is not well indented? 🤔

Reply:
Well, this is best effort. If the file is not well indented, we don't have any chance to fix it, right?

Reply:
Maybe calculating the average of the indentation could be more closer... but anyway, is up to you to do that effort.